lib/SILGen/SILGen.h

```diff
@@ -114,31 +114,6 @@ class LLVM_LIBRARY_VISIBILITY SILGenModule : public ASTVisitor<SILGenModule> {
 
   llvm::Optional<ProtocolConformance *> NSErrorConformanceToError;
 
-  llvm::Optional<FuncDecl *> AsyncLetStart;
```

Comment:
Looks like some of the preceding ones might be good candidates as well ;)

Reply:
Right, I'm planning to look into that separately! :)